packages/context/test/unit/resolver.test.ts

```diff
@@ -189,6 +231,24 @@ describe('property injection', () => {
     const t = instantiateClass(TestClass, ctx) as TestClass;
     expect(t.fooBar).to.eql('FOO:BAR');
   });
+
+  it('reports circular dependencies of two bindings', () => {
+    const context = new Context();
+    class XClass {
+      // tslint:disable-next-line:no-any
```

Comment:
Can you use a different type from `any` please, for example `Object`? I think the exact type should not really matter for this test, and it's always better to avoid tslint overrides.

Reply:
Fixed with interfaces.

Reply:
I am afraid this is not fixed - there are still `tslint:disable-next-line:no-any` comments in this test file.